src/test/scala/ReplaceRsvdKeytest.scala

```diff
@@ -1,20 +1,31 @@
+package essent
+import org.scalatest._
 import essent.passes.ReplaceRsvdKeywords
 import firrtl.CircuitState
 import firrtl.options.Dependency
 
 import java.io.{File, FileWriter}
 import scala.io.Source
 
-object ReplaceRsvdKeytest extends App {
-  val sourceReader = Source.fromFile(args(0))
-  val circuit = firrtl.Parser.parse(sourceReader.getLines, firrtl.Parser.IgnoreInfo)
-  sourceReader.close()
+class ReplaceRsvdKeytest extends FlatSpec{
+   "Mypass" should "Replace all reserve keyword" in {
+     val sourceReader = Source.fromURL(getClass.getResource("/Testrsvd.fir"))
+     val circuit = firrtl.Parser.parse(sourceReader.getLines, firrtl.Parser.IgnoreInfo)
+     sourceReader.close()
 
-  val deps = firrtl.stage.Forms.LowFormOptimized ++ Seq(Dependency(ReplaceRsvdKeywords))
-  val firrtlCompiler = new firrtl.stage.transforms.Compiler(deps)
-  val resultState = firrtlCompiler.execute(CircuitState(circuit, Seq()))
-  val debugWriter = new FileWriter(new File(s"${circuit.main}_transformed.fir"))
-  debugWriter.write(resultState.circuit.serialize)
-  debugWriter.close()
+     val deps = firrtl.stage.Forms.LowFormOptimized ++ Seq(Dependency(ReplaceRsvdKeywords))
+     val firrtlCompiler = new firrtl.stage.transforms.Compiler(deps)
+     val resultState = firrtlCompiler.execute(CircuitState(circuit, Seq()))
+     val tmpfile = File.createTempFile(s"${circuit.main}_transformed",".fir")
+     println(tmpfile)
+     val debugWriter = new FileWriter(tmpfile)
```

Comment:
The debugWriter may not be needed depending on how the result is checked for correctness.

Reply:
yes. I have removed that part.